### ISSUE TITLE
I has some problems with bottleneck_features.py

### ISSUE THREAD

**Issue by SUHAR1K** [MaxTitkov/Keras_InceptionV3_Binary_classification#1]
I downloaded a set of pictures of cats and dogs from kaggle.com, divided it and put them in img_train subfolders. Launched bottleneck_features.py then I got the error: `ValueError: could not broadcast input array from shape (32,2048,3,3) into shape (16,2048,3,3`. What could be the reason?

**Comment by MaxTitkov** [comment]
Hi. Try to delete this :
from keras import backend as K
K.set_image_dim_ordering('th')

**Comment by SUHAR1K** [comment]
I made it and got new error: `ValueError: The input must have 3 channels; got 'input_shape=(3, 150, 150)'`

**Comment by MaxTitkov** [comment]
Did you try to change the order of input shape?: 
inc_model=InceptionV3(include_top=False,
                      weights='imagenet',
                      input_shape=((150, 150, 3)))

**Comment by SUHAR1K** [comment]
Yes, i did. And got error: `ValueError: could not broadcast input array from shape (32,2048,3,3) into shape (16,2048,3,3` again (

**Comment by MaxTitkov** [comment]
I'll try to fix it on the week and write you

**Comment by SUHAR1K** [comment]
Ok, thanks!

**Comment by MaxTitkov** [comment]
Something strange. I've launched the code both the Ubuntu (Tensorflow backend) and Windows (with Theano backend) and it works. Please, try to upgrade your Keras via: ```pip install keras --upgrade``` and write me

**Comment by SUHAR1K** [comment]
When I updated kerras, I had an error `from six.moves import range`, I found a question about this error and it says that this is a bug in Ubuntu, use six==1.4.0. I updated Six and got error `AttributeError: type object 'NewBase' has no attribute 'is_abstract'`.
And I thought, maybe the problem is compatibility? You can write versions of your libraries and the OS you used in the Readme file?

**Comment by MaxTitkov** [comment]
I've added versions in readme. Do you use or not Anaconda?
